build.sbt

```diff
@@ -113,7 +113,7 @@ scalacOptions += "-Wconf:src=target/.*:s"
 
 scalacOptions ++= Seq("-unchecked", "-deprecation", "-feature")
 
-val bootstrapPlayVersion = "7.19.0"
+val bootstrapPlayVersion = "7.22.0"
```

Comment:
Pretty minor thing, but 7.23.0 was released last week so we should probably use that